apps/api/src/modules/library/dto/create-book.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateBookDto {
+    @Field({ nullable: true })
+    id?: number;
+
+    @Field()
+    title: string;
+
+    @Field()
+    author: string;
+
+    @Field()
+    userId: number;
```

Comment:
why would the frontend user pass the userId when it's included in the token? this can lead to vulnerabilities